cmd/heimdalld/cmd/root.go

```diff
@@ -320,8 +322,7 @@ func writeGenesisFile(genesisTime time.Time, genesisFile, chainID string, appSta
 
 // InitializeNodeValidatorFiles initializes node and priv validator files
 func InitializeNodeValidatorFiles(
```

Comment:
Are we using it anywhere with mnemonic input arg? I see a couple of functions calling with empty strings only. Let's not add this change if it not required.

Reply:
wqe need that , we checking https://github.com/maticnetwork/heimdall/pull/637/files#diff-fb94b32144fe4c699c27cf8d8ed4092e018fdd9f24c7a0aefbc20092ccedea62R345 there i